website/docs/r/glue_catalog_table.html.markdown

```diff
@@ -44,36 +44,38 @@ resource "aws_glue_catalog_table" "aws_glue_catalog_table" {
       name                  = "my-stream"
       serialization_library = "org.apache.hadoop.hive.ql.io.parquet.serde.ParquetHiveSerDe"
 
-      parameters {
+      parameters = {
         serialization.format = 1
```

Comment:
It doesn't seem to like `serialization.format`:
```
A managed resource "serialization" "format" has not been declared in pricing.
```
However `"serialization.format"` works as a key.

Reply:
Same issue with `parquet.compression` on line 35

Reply:
@marceloneil where is that error message coming from? I was using `terraform validate` with Terraform 0.12.1 to validate the syntax.

Before adding the equals sign (`=`) to tell the configuration language validator that this is a map argument, I was receiving:

```console
$ terraform validate

Error: Argument or block definition required

  on main.tf line 22, in resource "aws_glue_catalog_table" "aws_glue_catalog_table":
  22:         serialization.format = 1

An argument or block definition is required here. To set an argument, use the
equals sign "=" to introduce the argument value.
```

Afterwards it passed. Let me try applying it.

Reply:
Ah ha -- I was able to coerce the error with this full configuration:

```hcl
terraform {
  required_providers {
    aws = "2.13.0"
  }

  required_version = "0.12.1"
}

provider "aws" {
  region = "us-east-2"
}

resource "aws_glue_catalog_database" "example" {
  name = "example"
}

resource "aws_glue_catalog_table" "example" {
  name          = "example"
  database_name = "${aws_glue_catalog_database.example.name}"

  table_type = "EXTERNAL_TABLE"

  parameters = {
    EXTERNAL            = "TRUE"
    parquet.compression = "SNAPPY"
  }

  storage_descriptor {
    location      = "s3://my-bucket/event-streams/my-stream"
    input_format  = "org.apache.hadoop.hive.ql.io.parquet.MapredParquetInputFormat"
    output_format = "org.apache.hadoop.hive.ql.io.parquet.MapredParquetOutputFormat"

    ser_de_info {
      name                  = "my-stream"
      serialization_library = "org.apache.hadoop.hive.ql.io.parquet.serde.ParquetHiveSerDe"

      parameters = {
        serialization.format = 1
      }
    }

    columns {
      name = "my_string"
      type = "string"
    }

    columns {
      name = "my_double"
      type = "double"
    }

    columns {
      name    = "my_date"
      type    = "date"
      comment = ""
    }

    columns {
      name    = "my_bigint"
      type    = "bigint"
      comment = ""
    }

    columns {
      name    = "my_struct"
      type    = "struct<my_nested_string:string>"
      comment = ""
    }
  }
}
```

```console
$ terraform validate

Error: Reference to undeclared resource

  on main.tf line 25, in resource "aws_glue_catalog_table" "example":
  25:     parquet.compression = "SNAPPY"

A managed resource "parquet" "compression" has not been declared in the root
module.


Error: Reference to undeclared resource

  on main.tf line 38, in resource "aws_glue_catalog_table" "example":
  38:         serialization.format = 1

A managed resource "serialization" "format" has not been declared in the root
module.
```

Reply:
Fixed in 14fd76b56b101f8e922c77499b4adfe205d62e98 -- thanks for the heads up!!! 💯

Reply:
Perfect, thanks!